components/eventing-controller/pkg/handlers/utils.go

```diff
@@ -17,6 +20,13 @@ import (
 	"github.com/mitchellh/hashstructure"
 )
 
+// MessagingBackendHandler exposes a common handler interface for different messaging backend systems
+type MessagingBackendHandler interface {
```

Comment:
IMHO the postfix `Handler` is not needed. We are talking about _messaging backends_ and we are talking about the _BEB messaging backend_ and the _NATS messaging backend_.

Reply:
Done